### PR TITLE
Videoroom mute audio/video

### LINKED ISSUE #52: Mute/Unmute Videoroom's local tracks

### PR DESCRIPTION
**Pull request by Qizot**
Closes #52

### CODE REVIEW

**Review by mat-hek — CHANGES_REQUESTED** [review]

**Comment by mat-hek** [comment]
@Qizot we should also consider the case when someone wants to mute audio/video _before_ joining the room. We can make it an issue for now.

File: webrtc/videoroom/assets/src/room.ts [hunk 1]
```diff
@@ -1,6 +1,13 @@
 import "../css/app.scss";
 
-import { getRoomId, addVideoElement, removeVideoElement, setErrorMessage } from "./room_ui";
+import {
+  addVideoElement,
+  getRoomId,
+  removeVideoElement,
+  setErrorMessage,
+  setupMediaControls,
+  setupRoomUI,
```
Comment (mat-hek) [inline webrtc/videoroom/assets/src/room.ts:9]: you may consider merging these two

File: webrtc/videoroom/lib/videoroom_web/templates/room/index.html.eex [hunk 2]
```diff
@@ -2,8 +2,42 @@
...
+        </svg>
+    </div>
+    <div id="mic-off" class="ControlIcon" style="display: none;">
+        <svg fill="none" height="24" stroke="#fff" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+            <line x1="1" x2="23" y1="1" y2="23"/>
+            <path d="M9 9v3a3 3 0 0 0 5.12 2.12M15 9.34V4a3 3 0 0 0-5.94-.6"/>
+            <path d="M17 16.95A7 7 0 0 1 5 12v-2m14 0v2a7 7 0 0 1-.11 1.23"/>
+            <line x1="12" x2="12" y1="19" y2="23"/>
+            <line x1="8" x2="16" y1="23" y2="23"/>
+        </svg>
```
Comment (mat-hek) [inline webrtc/videoroom/lib/videoroom_web/templates/room/index.html.eex:41]: why are the SVGs inlined?

Comment (Qizot) [reply webrtc/videoroom/lib/videoroom_web/templates/room/index.html.eex]: I couldn't get it to work from files, fixed now.

**Review by mat-hek — APPROVED** [review]

**Pull request merged**